themes-default/slim/test/specs/snapshots/config.spec.js.md

```diff
@@ -0,0 +1,22 @@
+# Snapshot report for `test/specs/config.spec.js`
+
+The actual snapshot is saved in `config.spec.js.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## renders
+
+> Snapshot 1
+
+    `<div id="config-content"><table cellspacing="1" border="0" cellpadding="0" width="100%" class="infoTable"><tr><td><i class="icon16-config-application"></i> Medusa Info:</td><td>␊
+                    Branch:␊
+                    <span><a href="https://github.com/pymedusa/Medusa/tree/master" target="_blank" rel="noreferrer">master</a></span><br>␊
+                    Commit:␊
+                    <span><a href="https://github.com/pymedusa/Medusa/commit/f5973b28465246e67465d7d86bdc8aa74d13e659" target="_blank" rel="noreferrer">f5973b28465246e67465d7d86bdc8aa74d13e659</a></span><br>␊
+                    Version:␊
+                    <span><a href="https://github.com/pymedusa/Medusa/releases/tag/v0.2.8" target="_blank" rel="noreferrer">v0.2.8</a></span><br>␊
+                    Database:␊
+                    <span>44.11</span></td></tr><tr><td><i class="icon16-config-python"></i> Python Version:</td><td>2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)]</td></tr><tr><td><i class="icon16-config-ssl"></i> SSL Version:</td><td></td></tr><tr><td><i class="icon16-config-os"></i> OS:</td><td></td></tr><tr><td><i class="icon16-config-locale"></i> Locale:</td><td></td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="infoTableSeperator"><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td><i class="icon16-config-user"></i> User:</td><td></td></tr><tr><td><i class="icon16-config-dir"></i> Program Folder:</td><td></td></tr><tr><td><i class="icon16-config-config"></i> Config File:</td><td></td></tr><tr><td><i class="icon16-config-db"></i> Database File:</td><td></td></tr><tr><td><i class="icon16-config-cache"></i> Cache Folder:</td><td></td></tr><tr><td><i class="icon16-config-log"></i> Log Folder:</td><td></td></tr><tr><td><i class="icon16-config-arguments"></i> Arguments:</td><td><pre>[␊
+        "--nolaunch",␊
+        "--datadir=C:\\\\Medusa\\\\Data"␊
```

Comment:
@sharkykh didn't you fix this?

Reply:
No worries, just wanted to make sure in the case you had fixed it that it hadn't snuck back in.